ts/Extensions/OfflineExporting.ts

```diff
@@ -765,7 +766,8 @@ Chart.prototype.exportChartLocal = function (
                         { filename: chart.getFilename() },
                         options
                     ),
-                    fallbackToExportServer
+                    fallbackToExportServer,
+                    (): void => fireEvent(chart, 'exportChartLocalSuccess')
```

Comment:
Not sure about the naming - since `afterExportChartLocal` by convention would fire at the end of a function, and this is an async call in only one branch of execution? I would expect `afterExportChartLocal` to be a hook for wraps, while this is more of a traditional event signal that only happens in some situations. For example, I would expect the event named `afterExportChartLocal` to also fire after fallback to the export server, which this one does not. Thoughts?

Reply:
Okay, yes I'm with you.